test/CMakeLists.txt

```
@@ -193,6 +193,7 @@ make_test(SRC diff2.cpp)
 make_test(SRC dog.cpp)
 make_test(SRC dot.cpp)
 make_test(SRC empty.cpp)
+make_test(SRC event.cpp)
```

```suggestion
make_test(SRC event.cpp CXX11)
```
This will allow you to test the move constructors. by default we target C++ 98 in the tests.